pages/map.js

```diff
@@ -11,8 +11,9 @@ import mapboxgl from '!mapbox-gl'
 import styles from '@/styles/Map.module.css'
 
 import useHospitalDataByProvince from '@/hooks/useHospitalDataByProvince'
-
+import Select from 'react-select'
```

Comment:
i think we should re-use "search" component from main page to reduce bundle size

Reply:
good idea, but I think better change this after we implement Chakra UI in map page